Add anthos known issue for disk deletion and 2.5.1.3 RN (#834)
portworx/pxdocs: content/portworx-install-with-kubernetes/on-premise/anthos/_index.md

{{% content "shared/cloud-references-auto-disk-provisioning-vsphere-vsphere-install-common.md" %}}

#### Max storage nodes

> As part of Anthos cluster management operations like upgrades, cluster nodes will get recycled (deleted and re-created). During this process, the cluster will momentarily be scaled to more number of nodes than originally installed. For e.g if you had a 3 node cluster, you could have 4 nodes in the cluster while these management operations are going on.

Reviewer: ```suggestion
Anthos cluster management operations, such as upgrades, recycle cluster nodes by deleting and recreating them. During this process, the cluster momentarily scales up to more nodes than initially installed. For example, a 3-node cluster may increase to a 4-node cluster.
```

> To avoid PX creating storage on these additional nodes, we need to put a cap on the number of Portworx nodes that will act as storage nodes. We do so by setting max storage nodes zone.

Reviewer: ```suggestion
To prevent Portworx from creating storage on these additional nodes, you must cap the number of Portworx nodes that will act as storage nodes. You can do this by setting the `MAX_NUMBER_OF_NODES_PER_ZONE` environment variable according to the following formulae:
```

* If your Anthos cluster does not have zones configured, this number should be your initial number of cluster nodes
* If your Anthos cluster has zones configured, this number should be initial number of cluster nodes per zone

```text
export MAX_NUMBER_OF_NODES_PER_ZONE=3
```

> {{<info>}} In above command, 3 is an example. Change this as per your cluster.{{</info>}}

Reviewer: ```suggestion
{{<info>}} **NOTE:** In the command above, 3 is an example. Change this number to suit your cluster.{{</info>}}
```

```

{{% content "shared/portworx-install-with-kubernetes-4-apply-the-spec.md" %}}

## Known issues

### vSphere 6.5

> * **Issue**: If you are running on VMware vSphere  6.5 or lower, when you delete a worker node VM, all disks attached to the VM will get deleted. This is the default vSphere behavior. Deletion of the disks will cause Portworx cluster to loose quorum.

Reviewer: ```suggestion
* **Issue**: If you are running on VMware vSphere version 6.5 or lower: when you delete a worker node VM, all disks attached to the VM are deleted. While this is the default vSphere behavior, the deletion of these disks will cause your Portworx cluster to lose quorum.
```

> * **Workaround**: Upgrade to vSphere 6.7.3 and use Portworx 2.5.1.3 or higher

Reviewer: ```suggestion
* **Workaround**: Upgrade to vSphere 6.7.3 and use Portworx 2.5.1.3 or higher.
```

```text
export VER=$(kubectl version --short | awk -Fv '/Server Version: /{print $3}')
curl -fsL -o px-spec.yaml "https://install.portworx.com/{{% currentVersion %}}?kbver=$VER&mz=$MAX_NUMBER_OF_NODES_PER_ZONE&csida=true&c=portworx-demo-cluster&b=true&st=k8s&csi=true&vsp=true&ds=$VSPHERE_DATASTORE_PREFIX&vc=$VSPHERE_VCENTER&s=%22$VSPHERE_DISK_TEMPLATE%22&misc=-rt_opts%20kvdb_failover_timeout_in_mins=25"
```

{{% content "shared/portworx-install-with-kubernetes-4-apply-the-spec.md" %}}

## Known issues

### vSphere 6.5

> * **Issue**: If you are running on VMware vSphere  6.5 or lower, when you delete a worker node VM, all disks attached to the VM will get deleted. This is the default vSphere behavior. Deletion of the disks will cause Portworx cluster to loose quorum.

Reviewer: ```suggestion
* **Issue**: If you are running on VMware vSphere version 6.5 or lower: When you delete a worker node VM, vSphere deletes all disks attached to the VM. While this is the default vSphere behavior, the deletion of these disks will cause the Portworx cluster to lose quorum.
```
